src/test/java/io/github/jamsesso/jsonlogic/ReduceExpressionTests.java

```diff
@@ -17,6 +19,6 @@ public void testReduce() throws JsonLogicException {
     int[] data = new int[] {1, 2, 3, 4, 5, 6};
     Object result = jsonLogic.apply(json, data);
 
-    assertEquals(21.0, result);
+    assertEquals(BigDecimal.valueOf(21), result);
```

Comment:
Can we use same assertion style everywhere. This looks more readable.

Reply:
Converting to string checks
```java
assertEquals("2", result.toPlainString());
```
So that it can be more readable